React/Base/RCTConvert.m

```diff
@@ -375,6 +375,38 @@ + (NSLocale *)NSLocale:(id)json
     NSWritingDirectionNatural,
     integerValue)
 
++ (NSLineBreakStrategy)NSLineBreakStrategy:(id)json RCT_DYNAMIC
+{
+  static NSDictionary *mapping;
+  static dispatch_once_t onceToken;
+  dispatch_once(&onceToken, ^{
+    if (@available(iOS 14.0, *)) {
+      mapping = @{
+        @"none" : @(NSLineBreakStrategyNone),
+#if defined(__IPHONE_OS_VERSION_MAX_ALLOWED) && __IPHONE_OS_VERSION_MAX_ALLOWED >= 140000
```

Comment:
Comments from: @javache 

> I don't think this is required. iOS Apps are pretty much always compiled with the latest SDK if you want to submit them

Reply:
AFAIK, If this is not specified, a compile error occurs if the XCode version is lower.